course/models.py

```diff
@@ -685,6 +685,7 @@ class CourseModule(UrlMixin, models.Model):
     introduction = models.TextField(blank=True)
     course_instance = models.ForeignKey(CourseInstance, on_delete=models.CASCADE,
         related_name="course_modules")
+    reading_opening_time = models.DateTimeField(verbose_name=_("Opening time for reading"), null=True, blank=True)
```

Comment:
Is the verbose name of the field rendered in the form page? Or is there another keyword for the help string? The form page should display a help text that says that the reading opening time may be left empty if no material should be available before the module opening time.

Reply:
Done.